cmd/cored/disable_mockhsm.go

```diff
@@ -1,22 +1,17 @@
-//+build prod
+//+build disable_mockhsm
```

Comment:
Discussed out of band: no loopback authn will be considered a feature. The build tag will be `no_loopback_authn`.